### DISCUSSION TITLE
bluetooth headset's play/pause feature not working

### DISCUSSION THREAD

**Discussion by SaBenBurra** [i3/i3#5313] [Q&A] [answered]
## I'm submitting a…

<pre>
[x] Bug
[ ] Feature Request
[ ] Documentation Request
[ ] Other (Please describe in detail)
</pre>

## Current Behavior

My bluetooth headset's play/pause button works with gnome but not works when i switched to i3wm
## Expected Behavior

Buttons should be works normally
## Reproduction Instructions

Connect to a bluetooth headhphone and test the play/pause button
## Environment

Output of `i3 --moreversion 2>&-`:
<pre>
Binary i3 version:  4.21 (2022-09-21) © 2009 Michael Stapelberg and contributors

Running version: < 4.2-200
</pre>

<details><summary>Config file</summary><pre>
https://pastebin.com/ab7PvXv5
</pre>
</details>

<pre>
- Linux Distribution & Version: fedora36
- Are you using a compositor (e.g., xcompmgr or compton):
</pre>

**Comment by Airblader** [comment]
This isn't something the window manager is involved in whatsoever, so I'll convert this to a discussion.

**Reply by SaBenBurra** [reply]
But it only happens when i switched to i3
Not happens when using gnome

**Reply by Airblader** [reply]
i3 isn't a desktop environment. You're probably missing some DE feature here that gets disabled because you switched to another window manager. We have zero Bluetooth code in i3, nor do we interact with any system involved in that.

**Reply by Adar-Dagan** [reply]
@SaBenBurra Did you figure out what was the problem? I have the same issue

**Reply by SaBenBurra** [reply, marked as answer]
@Adar-Dagan install playerctl package and add this to your config file:
```
bindsym XF86AudioPlay exec playerctl play
bindsym XF86AudioPause exec playerctl pause
bindsym XF86AudioNext exec playerctl next
bindsym XF86AudioPrev exec playerctl previous
```
